frontend/package.json

```diff
@@ -9,15 +9,21 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@emotion/cache": "^11.11.0",
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.0",
+    "@mui/material-nextjs": "^5.15.11",
+    "@types/jest": "^29.5.12",
```

Comment:
Do we need `@types/jest`?

Reply:
In simple terms, @types/jest is a package that helps TypeScript understand and work better with Jest, which is a popular testing framework for JavaScript. It provides TypeScript with information about the types and interfaces used in Jest, making it easier to write and maintain tests in TypeScript. Essentially, it improves the development experience when using Jest with TypeScript by adding type safety and better tooling support.

Reply:
Can we remove it?